pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -43,7 +43,7 @@
     "Microsoft.DiaSymReader.Native": "1.4.1",
     "Libuv": "1.9.2",
     "runtime.native.System.IO.Compression": "4.3.1",
-    "runtime.native.System.Security.Cryptography": "4.3.2",
+    "runtime.native.System.Security.Cryptography": "4.3.3",
     "runtime.native.System.Security.Cryptography.Apple": "4.3.1",
     "runtime.native.System.Security.Cryptography.OpenSsl": "4.3.2"
```

Comment:
Please be sure to check any of the new package versions we are shipping to ensure they are explicitly listed here (assuming they are in Microsoft.NETCore.App).